manifests/zone.pp

```diff
@@ -28,6 +29,12 @@
 
   $zonefilename = "${zonefilepath}/${filename}"
 
+  if $zonetype == 'slave' {
+    $_dns_notify = pick($dns_notify, 'no')
```

Comment:
I recall at some point puppet doesn't like ```$_``` variables, but I'm not 100% sure about it.